include/ginkgo/core/matrix/csr.hpp

```diff
@@ -878,6 +966,18 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
         return values_.get_const_data();
     }
 
+    /**
+     * Creates a Dense view of the value array of this matrix as a column
+     * vector of dimensions nnz x 1.
+     */
+    std::unique_ptr<Dense<ValueType>> create_value_view();
```

Comment:
maybe keep those as free functions in our core?

Reply:
Where would be the right place to put them? I could create a new internal csr header, but maybe that's overkill?

Reply:
I thought they were only used in `csr.cpp`, but now I see that they are also used in tests, so I'm fine with them as it is.